app/jobs/school_email_update_job.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class SchoolEmailUpdateJob < CronJob
+  self.cron_expression = "30 6 * * *"
```

Comment:
I've chosen daily at 0630 since the source files seem to typically be generated by GIAS between 0530 and 0615